CASMINST-7008 Remove with IUF in main Upgrade CSM from Upgrade CSM and additional products with IUF (#5409)
Cray-HPE/docs-csm: upgrade/Stage_0_Prerequisites.md

There are several options for this stage. Use the option which applies to the current upgrade
scenario.

- [Option 1: Upgrade of CSM manually and additional products with IUF](#option-1-upgrade-of-csm-manually-and-additional-products-with-iuf)
- [Option 2: Upgrade of CSM on CSM-only system](#option-2-upgrade-of-csm-manually-on-csm-only-system)

### Option 1: Upgrade of CSM manually and additional products with IUF

> If performing an upgrade of CSM manually and additional HPE Cray EX software products with IUF, this stage
> should not be performed. Instead, the [Upgrade CSM manually and additional products with IUF](../operations/iuf/workflows/upgrade_csm_manual_and_additional_products_with_iuf.md)

Reviewer: ```suggestion
This stage should not be performed when upgrading CSM manually and additional HPE Cray EX software products with IUF. Instead, refer to the [Upgrade CSM manually and additional products with IUF](../operations/iuf/workflows/upgrade_csm_manual_and_additional_products_with_iuf.md) procedure.
```
